cabal.project

```diff
@@ -64,8 +63,17 @@ source-repository-package
 source-repository-package
   type: git
   location: https://github.com/geniusyield/atlas
-  tag: be490dcc5b99d0ff10e9d7cd801c4e0ae3a4865a
-  --sha256: sha256-HxwD8p7eg1yMo3hVDtGduU9zOQobFPONKj+064fdjVU=
+  tag: 105ff78b32935ae5c18c53ccf4fb8462b8c1ca03
+  --sha256: sha256-Ee/UIoJ8OYBxgdnSv61hN8DH7CrIOPn4DjqOBxFL3Js=
+
+source-repository-package
+  type: git
+  location: https://github.com/geniusyield/dex-contracts-api
```

Comment:
Using commit from [this](https://github.com/geniusyield/dex-contracts-api/pull/40) PR. 
Let's merge PR at `dex-contracts-api` side once our systems are all up to date, since that PR would create orders at v1.1 script.